packages/theme/src/cli/utilities/theme-environment/theme-polling.ts

```diff
@@ -45,62 +45,84 @@ export async function pollRemoteJsonChanges(
   localFileSystem: ThemeFileSystem,
   options: PollingOptions,
 ): Promise<Checksum[]> {
-  const filteredRemoteChecksums = await applyFileFilters(remoteChecksums, localFileSystem, options)
-
+  const previousChecksums = await applyFileFilters(remoteChecksums, localFileSystem, options)
   const latestChecksums = await fetchChecksums(targetTheme.id, currentSession).then((checksums) =>
     applyFileFilters(checksums, localFileSystem, options),
   )
 
-  const previousChecksumsMap = new Map(filteredRemoteChecksums.map((checksum) => [checksum.key, checksum]))
+  const assetsChangedOnRemote = getChangedAssets(previousChecksums, latestChecksums)
```

Comment:
What do you think if we rename the variables/methods to be more specific in the method name and less specific in the variable name? In other words, we’d have something like this here:

```ts
const changedAssets = getAssetsChangedOnRemote(previousChecksums, latestChecksums)
const deletedAssets = getAssetsDeletedFromRemote(latestChecksums, previousChecksums)
```

(I believe that being more specific in the method name may be more expressive for future readers, as we clarify the meaning of that method)